### ISSUE TITLE
Message being cut off

### ISSUE THREAD

**Issue by msl2000** [UXDivers/Grial-UI-Kit-Support#406]
Hi, for some odd reason, some long messages (not all) gets cut off when being displayed in the chat list.

as shown below.

![image](https://user-images.githubusercontent.com/18165861/38589424-36d2b2f4-3d6e-11e8-83e5-a4e33ce530ce.png)

**Comment by jpbrocca** [comment]
Hi @msl2000,

That is weird, looking at the sample app it seems messages properly grow in height.

Did you changed anything from the original sample?

Check if your `ListView` has the property `HasUnevenRows` set to `true`.

Best
JP

**Comment by msl2000** [comment]
Hi JP

aside from adding a name to the listview, nothing. Here is another image with a long message and its fine here. The problem seems to happen randomly to some messages. Especially if the message just sort of exceeds 1 line

![image](https://user-images.githubusercontent.com/18165861/38650527-b7aa3d6c-3e3f-11e8-944d-b752213789bf.png)

**Comment by msl2000** [comment]
Hi @jpbrocca. Found the issue. Its mainly on the right chat side. When you have a sentence that just slightly exceeds one line, this issue occurs. If its a really long, the issue wont occur. Seems to be the boxview refusing to enlarge based on the text's length.

Fix I did was basically replacing boxview with a stacklayout instead and putting the message label inside like so.

```
<!-- BALLOON BACKGROUND -->
        <StackLayout Grid.Row="0" Grid.Column="0" BackgroundColor="{ DynamicResource ChatRightBalloonBackgroundColor }" VerticalOptions="FillAndExpand">
            <Label Margin="10" TextColor="{ DynamicResource ChatRightTextColor }" FontSize="12" Text="{ Binding message }" VerticalOptions="EndAndExpand" />
        </StackLayout>
```

**Comment by msl2000** [comment]
No, there was no tapping involved. The message just got cut off because the boxview didn't resize according to the label's text length.
